docs/docsite/rst/community/development_process.rst

```diff
@@ -41,8 +41,8 @@ more likely it will be that a PR that reaches “shipit” will be
 mergeable.
 
 .. _Ansibull PR Triage Bot: https://github.com/ansible/ansibullbot/blob/master/triage.py
-.. _Core: https://github.com/ansible/ansibullbot/blob/master/MAINTAINERS-CORE.txt
-.. _Extras: https://github.com/ansible/ansibullbot/blob/master/MAINTAINERS-CORE.txt
+.. _Core: https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml
+.. _Extras: https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml
```

Comment:
So there's no more split between Core and Extras, I wonder if we still want to pretend there is.